### PR TITLE
Usage of className in BpkAccordionItem replaced with wrapper spans

### PR DESCRIPTION
**Pull request by Sybsw**
Placing usage of className prop in two internal components with spans set to the same style classes

Remember to include the following changes:

- [x] Ensure the PR title includes the name of the component you are changing so it's clear in the release notes for consumers of the changes in the version e.g `[KOA-123][BpkButton] Updating the colour`
- [ ] `README.md` (If you have created a new component)
- [ ] Component `README.md`
- [x] Tests
- [ ] Storybook examples created/updated
- [ ] Type declaration (`.d.ts`) files updated
- [ ] For breaking changes or deprecating components/properties, migration guides added to the description of the PR. If the guide has large changes, consider creating a new Markdown page inside the component's docs folder and link it here

### CODE REVIEW

**Comment by olliecurtis** [comment]
Can you ensure the PR title includes 

[JIRA Number][Component updated] as per the first item in the checklist

**Review by olliecurtis — COMMENTED** [review]
Can you ensure the PR title includes

[JIRA Number][Component updated] as per the first item in the checklist

File: packages/bpk-component-accordion/src/BpkAccordionItem.js [hunk 1]
```diff
@@ -134,16 +134,19 @@ const BpkAccordionItem = (props: Props) => {
...
+                textStyle={textStyle}
+                tagName={tagName}
+              >
+                {clonedIcon}
+                {title}
+              </BpkText>
+            </span>
             <span className={getClassName('bpk-accordion__icon-wrapper')}>
-              <ExpandIcon className={iconClassNames.join(' ')} />
+              <div className={iconClassNames.join(' ')}>
```
Comment (runmoore) [inline packages/bpk-component-accordion/src/BpkAccordionItem.js:147]: be careful when putting `div` tags inside of spans. The HTML spec likely doesn't allow it, https://stackoverflow.com/questions/11314668/what-elements-can-a-span-tag-contain-in-html5

Comment (Sybsw) [reply packages/bpk-component-accordion/src/BpkAccordionItem.js:147]: Replaced with a div, no negative effects except the already happening vertical space problem

**Comment by Sybsw** [comment]
Closed for

**Pull request closed without merging**